styles/light/_light-theme.scss

```diff
@@ -518,10 +518,6 @@ pre {
 
 // Homepage
 .page-home {
-  #home-hero-graphic {
-    content: url("../img/lightmode/home-hero.svg");
-  }
-
```

Comment:
Currently, the home hero image has two versions. The version for light mode has slightly darker colors for the "X" to improve contrast. (Compare: [for dark mode](https://xrpl.org/img/home-hero.svg) vs [for light mode](https://xrpl.org/img/lightmode/home-hero.svg))

I don't know if this change was intentional or accidental. If it's intentional, then you should also delete the light mode image now that it's not used.